docs/index.md

```diff
@@ -1,5 +1,54 @@
-# Cal-ITP websites
+# Project overview
```

Comment:
I think I'd probably leave this H1 as "Cal-ITP websites"? We might think of them as a single project, but that sorta seems like the wrong way to frame these docs.

Reply:
makes sense to me. I was mostly just trying to avoid the duplication with the mermaid title but once we add some more content to this page that'll be moot anyway.